scripts/netwarn.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
Many possible visual + other improvements here:

https://github.com/iiab/iiab/pull/3308#issuecomment-1187905693

Reply:
read the comment in the commit.

Reply:
https://github.com/iiab/iiab/pull/3308#issuecomment-1187761321

Reply:
https://github.com/iiab/iiab/pull/3308#issuecomment-1188353462

Reply:
Let me reiterate:

- https://github.com/iiab/iiab/pull/3308#issuecomment-1187905693 adds many improvements for OS's that do not permit bash during autostart.  (Apparently Ubuntu's LightDM being such a situation, there may be others!)

- Whatever is or is not on the top 1-2 lines is `/usr/local/sbin/netwarn.sh` is extraneous.  (What should appear on the top 1-2 lines of netwarn.sh, is any comment that explains this unusual lack of a shebang line [or an eccentric shebang line or whatever's appropriate] and the specific underlying reasons.)